lib/index.js

```diff
@@ -43,4 +43,5 @@ ZAFClient.init = function(callback, loc) {
   return client;
 };
 
+window.ZAFClient = ZAFClient;
```

Comment:
Shouldn't we be using https://webpack.js.org/configuration/output/#output-library?

Reply:
I would love to use it, but I can't get it to work.